CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.2
```

Comment:
Looks like @domesticmouse  beat you to this version number :)

Reply:
Sorry about that, just published =)

Reply:
No worries.  Rebased and bumped to 0.7.3